packages/fast-form-generator-react/src/form/form-item.textarea.style.ts

```diff
@@ -17,7 +17,9 @@ const styles: ComponentStyles<IFormItemTextareaClassNameContract, {}> = {
     formItemTextarea_textarea: {
         ...applyInputStyle(),
         marginTop: toPx(8),
-        marginBottom: toPx(2)
+        marginBottom: toPx(2),
+        resize: "none",
+        fontFamily: "Segoe UI, SegoeUI, Helvetica Neue, Helvetica, Arial, sans-serif"
```

Comment:
Does "inherit" not work here? If it doesn't, we should consider setting a variable somewhere and importing that. Having multiple instances of these strings throughout the package will prove difficult to maintain.

Reply:
yes "inherit" does work.